src/app/movie-details/movie-details.component.html

```diff
@@ -0,0 +1,13 @@
+<button (click)="getTitles()">Details</button>
+<div *ngIf="data">
+    <h3>{{data.title}}</h3> 
+    <img  src="{{imagePath}}{{data.poster_path}}">
+    <li><b>Date:</b> {{data.release_date}}</li>
+    <li><b>Overview:</b>{{data.overview}}</li>
+    <li><b>Genres:</b></li> 
+</div>
+
+<div *ngFor="let m of genres">
```

Comment:
jeśli masz `genres` to powinno być `let genre of genres` - nie narzucone, ale dobra praktyka